lib/LightPanel/LightPanel.js

```diff
@@ -149,12 +149,23 @@ module.exports = kind(
 		if (!this.$.client.children.length) {
 			this.createComponents(this.clientComponents);
 			this.$.client.render();
-			this.$.client.addClass('populated');
+			this.didClientRender();
 		}
 
-		if (!Spotlight.getCurrent()) {
+		if (!Spotlight.getPointerMode() && !Spotlight.isSpottable(Spotlight.getCurrent(), true)) {
```

Comment:
Then, I think it will be better to just add didClientRender in this PR. I'll change that code.